Machines/machine_postprocess.xml

```diff
@@ -62,6 +62,12 @@
 	<initialize_pes queue="geyser" pes_per_node="1" wallclock="00:10">1</initialize_pes>
 	<obs_root>/glade/p/cesm/lmwg_dev/oleson/ILAMB/ILAMB_all</obs_root>
       </component>
+      <component name="iomb">
+	<averages_pes queue="regular" pes_per_node="0" wallclock="00:01">0</averages_pes> <!-- unsued. not required by xsd but required by the code. -->
+	<diagnostics_pes queue="geyser" pes_per_node="2" wallclock="01:00">2</diagnostics_pes>
+	<initialize_pes queue="geyser" pes_per_node="1" wallclock="00:10">1</initialize_pes>
+	<obs_root>/glade/p/cesm/omwg/obs_data/IOMB/IOMB_all</obs_root>
```

Comment:
This can just be /glade/p/cesm/omwg/obs_data/IOMB

Reply:
Fixed in ae7e41c.

Reply:
We also need to update and test for cheyenne.